CMakeLists.txt

```diff
@@ -192,6 +196,30 @@ endif()
 # Build Tests and download Catch2
 if (${PARTHENON_ENABLE_UNIT_TESTS} OR ${PARTHENON_ENABLE_INTEGRATION_TESTS} OR ${PARTHENON_ENABLE_REGRESSION_TESTS} OR ${PARTHENON_ENABLE_PERFORMANCE_TESTS})
 
+  # if running regression test automatically synch gold standard
+  if (${PARTHENON_ENABLE_REGRESSION_TESTS} AND ${REGRESSION_GOLD_STANDARD_SYNC})
+    if (EXISTS ${PROJECT_SOURCE_DIR}/tst/regression/gold_standard/current_version)
+      file(READ ${PROJECT_SOURCE_DIR}/tst/regression/gold_standard/current_version REGRESSION_GOLD_STANDARD_ON_DISK)
+    else()
+      set(REGRESSION_GOLD_STANDARD_ON_DISK 0)
+    endif()
+    if (NOT (${REGRESSION_GOLD_STANDARD_ON_DISK} EQUAL ${REGRESSION_GOLD_STANDARD}))
+      ExternalProject_Add(
+        gold_standard
+        URL https://github.com/lanl/parthenon/releases/download/regression-gold-v${REGRESSION_GOLD_STANDARD}/parthenon_regression_gold_v${REGRESSION_GOLD_STANDARD}.tgz
+        URL_HASH SHA512=45f57d16b76a3a44940e40ea642c1d5a2b3aea681a3064e4c06b031bf264d627f8009e967b92a2cb0fbc08e5cd0ffda381038483f390fa00f47c42a551ca3646
+        SOURCE_DIR "${PROJECT_SOURCE_DIR}/tst/regression/gold_standard"
+        DOWNLOAD_DIR "${PROJECT_SOURCE_DIR}/tst/regression/gold_standard"
+        # just need the extracted data so no configure/build/install
+        CONFIGURE_COMMAND ""
+        BUILD_COMMAND ""
+        INSTALL_COMMAND ""
```

Comment:
This is neat, I didn't know about the ExternalProject cmake module.

Reply:
In some way, if you don't run configure/build/install, `FetchContent_Declare()` or even `file(DOWNLOAD..)` might be the more natural thing to do.

Reply:
Good point. I'm now directly using the `file` features.
At some point it may be worth thinking about using this for Kokkos and catch2 (instead of cloning the entire repos as submodules).